### DISCUSSION TITLE
Can you flash back to the prusa firmware after flashing klipper?

### DISCUSSION THREAD

**Discussion by VACInc** [dz0ny/klipper-prusa-mk3s#14] [Q&A] [answered]
Curious if I can just plug in my PC and flash the latest prusa firmware after flashing klipper if I decide to revert back?

**Comment by dz0ny** [comment, marked as answer]
Yes, you can flash back the stock firmware, it will even retain your config data as that is stored in EEPROM, which Klipper does not use.
